device-types/DataDirect Networks/SFA200NVX2-Enclosure.yaml

```diff
@@ -0,0 +1,72 @@
+---
+manufacturer: DataDirect Networks
+model: SFA200NVX2 Enclosure
+slug: datadirect-networks-sfa200nvx2-enclosure
+comments: 24bay Enclosure for SFA200NVX2 or SFA400NVX2
+part_number: ENC-200NVX2
+u_height: 2
+is_full_depth: true
+airflow: front-to-rear
+subdevice_role: parent
+module-bays:
+  - name: Controller 0
+    label: Controller Slot 0
+    position: '0'
+  - name: Controller 1
+    label: Controller Slot 1
+    position: '1'
+  - name: PSU 1
+    label: PSU Slot 1
+    position: '1'
+  - name: PSU 2
+    label: PSU Slot 2
+    position: '2'
+device-bays:
+  - name: '1'
+    label: NVMe Slot 1
+  - name: '2'
+    label: NVMe Slot 2
+  - name: '3'
+    label: NVMe Slot 3
+  - name: '4'
+    label: NVMe Slot 4
+  - name: '5'
+    label: NVMe Slot 5
+  - name: '6'
+    label: NVMe Slot 6
+  - name: '7'
+    label: NVMe Slot 7
+  - name: '8'
+    label: NVMe Slot 8
+  - name: '9'
+    label: NVMe Slot 9
+  - name: '10'
+    label: NVMe Slot 10
+  - name: '11'
+    label: NVMe Slot 11
+  - name: '12'
+    label: NVMe Slot 12
+  - name: '13'
+    label: NVMe Slot 13
+  - name: '14'
+    label: NVMe Slot 14
+  - name: '15'
+    label: NVMe Slot 15
+  - name: '16'
+    label: NVMe Slot 16
+  - name: '17'
+    label: NVMe Slot 17
+  - name: '18'
+    label: NVMe Slot 18
+  - name: '19'
+    label: NVMe Slot 19
+  - name: '20'
+    label: NVMe Slot 20
+  - name: '21'
+    label: NVMe Slot 21
+  - name: '22'
+    label: NVMe Slot 22
+  - name: '23'
+    label: NVMe Slot 23
+  - name: '24'
+    label: NVMe Slot 24
```

Comment:
NetBox is not currently setup to handle storage management. Please remove these definitions from this file

We are working on a storage solution but it is not ready and `device-bays` is not meant for storage devices.

Reply:
ok